python-dict-attribute/config_v2.py

```diff
@@ -0,0 +1,21 @@
+class Config:
+    def __init__(self, name):
+        self.name = name
+
+    def set_option(self, key, value):
+        setattr(self, key, value)
+
+    def get_option(self, key):
+        return getattr(self, key, None)
+
+    def remove_option(self, key):
+        if hasattr(self, key):
+            delattr(self, key)
+            print(f"'{key}' removed!")
+        else:
+            print(f"'{key}' does not exist.")
+
+    def clear(self):
+        for key in list(self.__dict__.keys()):
+            delattr(self, key)
+        print("All options removed!")
```

Comment:
Optionally, you could add the print calls at the bottom of all of these config files, so that they produce a result when learners run them:

```suggestion
        print("All options removed!")

config = Config(theme="light", font_size=12, language="English")
print(config)

user = {"theme": "dark", "font_size": 14, "language": "Spanish"}
config.update(**user)
print(config)
```